sqltest/testdata/planner_index_test_script.txt

```diff
@@ -0,0 +1,33 @@
+--create topic testtopic;
+use test;
+create source transactions(
+    id bigint,
+    customer_id tinyint,
+    col2 varchar,
+    primary key (id)
+) with (
+    brokername = "testbroker",
+    topicname = "testtopic",
+    headerencoding = "json",
+    keyencoding = "json",
+    valueencoding = "json",
+    columnselectors = (
+        meta("key").k0,
+        v1,
+        v2
+    )
+);
+
+create index index1 on transactions (customer_id);
+
+--load data dataset_1;
+
+--test with source;
+
+select customer_id from transactions where customer_id = 600;
```

Comment:
It would be good to also test cases where:
1. customer_id is null
2. with a value of customer_id that returns multiple rows
3. with a value of customer_id which doesn't match any rows